CONTRIBUTING.md

```diff
@@ -10,19 +10,22 @@ Here's a quick guide for contributing:
 
 1. Fork the repo.
 
-2. Run the tests. We only take pull requests with passing tests, and it's great
+2. Make sure you have ImageMagick and Ghostscript installed. See [this section]
+(./README.md#image-processor) of the README.
+
+3. Run the tests. We only take pull requests with passing tests, and it's great
 to know that you have a clean slate: `bundle && bundle exec rake`
 
-3. Add a test for your change. Only refactoring and documentation changes
+4. Add a test for your change. Only refactoring and documentation changes
 require no new tests. If you are adding functionality or fixing a bug, we need
 a test!
 
-4. Make the test pass.
+5. Make the test pass.
 
-5. Mention how your changes affect the project to other developers and users in
+6. Mention how your changes affect the project to other developers and users in
    the `NEWS.md` file.
 
-6. Push to your fork and submit a pull request.
+7. Push to your fork and submit a pull request.
```

Comment:
Let's make all these numbers `1`. Markdown's generated `<ol>` will take care of assigning the right number, and we don't pollute the diff when we change the order or add/remove steps.